packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -324,6 +324,38 @@ export function getPublishResolver<TSource, TContext, TArgs>({
   }
 }
 
+/**
+ *  Returns values for link arguments, also covers the cases for
+ * if the link parameter contains constants that are appended to the link parameter
+ *
+ * e.g. instead of:
+ * $response.body#/employerId
+ *
+ * it could be:
+ * abc_{$response.body#/employerId}
+ */
+function inferLinkArguments (paramName, value, resolveData, source, args) {
+  if (Object.prototype.toString.call(value) === '[object Object]') {
+    return Object.entries(value).reduce((acc, [key, value]) => {
```

Comment:
Thanks for fixing this issue as well! However, I think we are missing a test for this and while I was trying to add it in, I came across some other issues...